### DISCUSSION TITLE
Profile Achievement Progress

### DISCUSSION THREAD

**Discussion by Glazelf** [community/community#19151] [Other Feature Feedback, Questions, & Ideas]
Namely for the Pull Shark achievement I think it would be beneficial if there was a more detailed progress note than the level.
Especially seeing the gap between level 3 and 4 being ~900 PRs, it would be usefull to have a progress bar or counter seperate from the level to indicate progress to the next level or how many PRs you have total.

This may also apply to different/future achievements, Galaxy Brain has smaller steps but I feel like this would still be a good change.

**Comment by D7EAD** [comment]
I agree! A feature such as a hover-over dialog that displays your progress to the next tier would be great!

**Comment by daven97** [comment]
I would love this feature as well! I think this would be nice for the Pair Extraordinaire badge!

**Comment by 3kh0** [comment]
This can be helpful for starstruck and galaxy brain as well.

**Comment by GutZuFusss** [comment]
is this still on the radar?

**Comment by nicosammito** [comment]
Any updates on that?

**Comment by ahampal** [comment]
would be nice to see a progress bar

**Comment by yossibaruch7** [comment]
> would be nice to see a progress bar

**Comment by yossibaruch7** [comment]
> Any updates on that?

**Comment by yossibaruch7** [comment]
> > would be nice to see a progress bar
> 
>
